osgbuild/mock.py

```diff
@@ -124,6 +126,23 @@ def _init_get_cfg_path(self):
                 os.path.join(cfg_dir,"mock-koji-%s-%s.%d.cfg" % (mock_config_from_koji, arch, os.getuid())),
                 mock_config_from_koji,
                 distro_tag)
+            # Can't really do this via the command line so hack up the mock config.
+            # This is not very safe -- the mock config is actually a Python script.
+            # TODO Probably better fixed as a Koji patch
+            cfg_path_orig = cfg_path + ".orig"
+            shutil.copy(cfg_path, cfg_path_orig)
+            with open(cfg_path_orig, "r") as cfg_in, open(cfg_path, "w") as cfg_out:
+                for line in cfg_in:
+                    # change the line
+                    #  config_opts['yum.conf'] = '...[build]\n...\n'
+                    # to
+                    #  config_opts['yum.conf'] = '...[build]\n...\nmodule_hotfixes=1\n'
+                    if line.startswith("config_opts['yum.conf']") and "[build]" in line and "module_hotfixes" not in line:
+                        new_line = re.sub(r"\\n'$", r"\\nmodule_hotfixes=1\\n'", line)
```

Comment:
my head hurts.

The substitution here is with the `\` + `n` sequences, right?  Not the actual newlines in the cfg files?

Reply:
ayup, there are literally `\n` sequences in the config file